src/main/java/com/mraof/minestuck/skaianet/ComputerWaitingList.java

```diff
@@ -115,8 +115,8 @@ void validate(MinecraftServer mcServer)
 			Map.Entry<PlayerIdentifier, ComputerReference> data = i.next();
 			ISburbComputer computer = data.getValue().getComputer(mcServer);
 			if(computer == null || data.getValue().isInNether() || !computer.getOwner().equals(data.getKey())
-					|| !(forClients && computer.getClientBoolean("isResuming")
-					|| !forClients && computer.getServerBoolean("isOpen")))
+					|| !(forClients && computer.getProgramBoolean(0, "isResuming")
+					|| !forClients && computer.getProgramBoolean(1, "isOpen")))
```

Comment:
This is not great for readability as it is not visible here which program is for which id. I recommend either reverting and look for some other way of approaching the problem, or introducing named constants somewhere to use instead of raw id numbers.